README.md

```diff
@@ -29,12 +28,20 @@ pip3 install ulozto-downloader
 ## Použití
 
 ```shell
-ulozto-downloader --parts 15 'https://ulozto.cz/file/TKvQVDFBEhtL/debian-9-6-0-amd64-netinst-iso'
+ulozto-downloader --parts 15 "https://ulozto.cz/file/TKvQVDFBEhtL/debian-9-6-0-amd64-netinst-iso"
+```
+
+Pro volbu manuálního či automatického čtení captcha kódů slouží přepínač *captcha* s hodnotami "m" či "a" (manuální je default).
+Abychom nemuseli captcha kódy sami luštit, můžeme tedy zavolat například
+
+```shell
+ulozto-downloader --parts 15 "https://ulozto.cz/file/TKvQVDFBEhtL/debian-9-6-0-amd64-netinst-iso" --captcha "a"
 ```
 
 ## Requirements
```

Comment:
Myslím že tato sekce by se už mohla rovnou vypustit, vzhledem k tomu že pip vše instaluje automaticky a tato sekce budí dojem že ne (a kdo si to chce stáhnout manuálně tak typicky chápe že má udělat pip install -r requirements.txt)

Reply:
Upraveno